### ISSUE TITLE
Wrong argument in API requests

### ISSUE THREAD

**Issue by kelson42** [openzim/mwoffliner#783]
```
[info] [2019-06-06T17:03:59.887Z] Getting JSON from [https://en.wikivoyage.org/w/api.php?action=query&format=json&prop=coordinates%7Crevisions%7Credirects&generator=allpages&gapfilterredir=nonredirects&gaplimit=max&gapnamespace=0&rawcontinue=true&rdlimit=max&clshow=!hidden&gapcontinue=Island_Park&cocontinue=16047%7C36457954]
[warn] [2019-06-06T17:04:00.396Z] Got warning from MW Query {
	"main": {
		"*": "Unrecognized parameter: clshow."
	}
}
[info] [2019-06-06T17:04:00.415Z] Getting JSON from [https://en.wikivoyage.org/w/api.php?action=query&format=json&prop=coordinates%7Crevisions%7Credirects&generator=allpages&gapfilterredir=nonredirects&gaplimit=max&gapnamespace=0&rawcontinue=true&rdlimit=max&clshow=!hidden&gapcontinue=Island_Park&cocontinue=16069%7C36440913]
[warn] [2019-06-06T17:04:01.220Z] Got warning from MW Query {
	"main": {
		"*": "Unrecognized parameter: clshow."
	}
}
```

**Comment by kelson42** [comment]
Tested with 1.9.3 rc3 and we still have one warning displayed
```
[log] [2019-06-11T13:34:35.790Z] Worker [0] getting article range [0-50] of [1] [5000%]
[info] [2019-06-11T13:34:35.791Z] Getting JSON from [https://pl.wikibooks.org/w/api.php?titles=Wikibooks%3AStrona_g%C5%82%C3%B3wna&prop=redirects%7Crevisions%7Cpageimages%7Ccoordinates&action=query&format=json&rdlimit=max&colimit=1]
[warn] [2019-06-11T13:34:35.981Z] Got warning from MW Query {
	"main": {
		"*": "Unrecognized parameter: colimit."
	},
	"query": {
		"*": "Unrecognized values for parameter \"prop\": pageimages, coordinates."
	}
}
[info] [2019-06-11T13:34:35.981Z] Got a warning when fetching coordinates, will not try again for any article
```

Comments about last PR says everything.

**Comment by ISNIT0** [comment]
`Got a warning when fetching coordinates, will not try again for any article`

**Comment by kelson42** [comment]
@ISNIT0 We should have *any* warning. There nothing to warn about here.

**Comment by kelson42** [comment]
Still there. I don't understand why this ticket has been closed:
```
[info] [2019-07-21T12:08:42.120Z] Getting JSON from [https://hitchwiki.org/en/api.php?action=query&format=json&prop=redirects%7Crevisions%7Cpageimages%7Ccoordinates&rdlimit=max&rdnamespace=0&titles=Main_Page&colimit=max]
[warn] [2019-07-21T12:08:42.245Z] Got warning from MW Query {
	"main": {
		"*": "Unrecognized parameter: colimit."
	},
	"query": {
		"*": "Unrecognized values for parameter \"prop\": pageimages, coordinates."
	}
}
```

**Comment by ISNIT0** [comment]
This was resolved 10 days ago, but 1.9.4 was released 14 days ago. So I wouldn't expect it to be fixed in 1.9.4
